packages/sdk/tasks/one-time/jarvis-bsc-fix.ts

```diff
@@ -0,0 +1,140 @@
+/**
```

Comment:
these one-time files are going to be removed after the deploy or refactored for general usage?

Reply:
probably just removed after use. Maybe kept for reference. It just now the most feasible way to get the pool up and understanding what the process is to put it into place in the frontend somehow.